loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowResult.java

```diff
@@ -6,18 +6,76 @@
  */
 package com.powsybl.loadflow;
 
+import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 
 /**
+ * Loadflow result API.
  *
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 public interface LoadFlowResult {
 
+    /**
+     * Result for one component
+     */
+    interface ComponentResult {
+
+        /**
+         * Get component number.
+         * @return component number
+         */
+        int getComponentNum();
+
+        /**
+         * Get detailed status of the computation on this component.
+         * @return the detailed status of the computation on this component
+         */
+        String getStatus();
```

Comment:
I wonder about the possibility to have an enum here with at least:
- CONVERGED
- MAX_ITERATION_REACHED
- SOLVER_FAILED: I am not sure about the meaning
- NO_CALCULATION
- Hades2 has a status that says if the result is realist or not. I think we don't need it because limit violations detection should be in the post processing.

Reply:
I would keep a string as status set could be different between loadflow implementations

Reply:
Thinking again, maybe you are right... if we want to rely on status to build more high level workflows we have to define all possible statuses.

Reply:
Result not realist means we have converged to a state with very low voltages so outside by default [0.8, 1.2]. I think we should keep it.